cmake/cpp.cmake

```diff
@@ -279,6 +279,12 @@ install(
   "${PROJECT_BINARY_DIR}/${PROJECT_NAME}ConfigVersion.cmake"
   DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}"
   COMPONENT Devel)
+install(
```

Comment:
can you guard it with a `if(USE_COINOR)`, i.e. if the coin-or support is disable there is no interest to install module